src/ArcGISEarth.AutoAPI.Utils/ArcGISEarth.AutoAPI.Utils.csproj

```diff
@@ -43,14 +44,15 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="EarthNamedpipeAPIUtils.cs" />
-    <Compile Include="IEarthNamedpipeService.cs" />
-    <Compile Include="ProcessUtils.cs" />
+    <Compile Include="AutomationAPIHelper.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json">
-      <Version>11.0.2</Version>
+      <Version>12.0.3</Version>
+    </PackageReference>
+    <PackageReference Include="RestSharp">
+      <Version>106.11.5-alpha.0.2</Version>
```

Comment:
The New 3rd party library needs to be reviewed by License team.  And you'd better use a normal release version, not a an alpha version.